src/components/InputTodo.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+
+const InputTodo = (props) => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (inputText.title.trim()) {
+      props.addTodoProps(inputText.title);
+      setInputText({
+        title: '',
+      });
+    } else {
+      alert('Please write item');
```

Comment:
`alert()` function will bring up an eslint warning when it runs a check through your code.
Using this function goes against JavaScript best practices, because it stops code execution and is limited in functionality. It is also a bad design and makes your page appear less attractive. 
Consider using a modal dialog instead.

Reply:
Removed the alert and replaced it with JavaScript best practice at this [commit](https://github.com/Lucash2022/react-todo-app/commit/a81c71cd96c71dcbacdcb2adfea51b58620f7202)
https://www.loom.com/share/c51f3602f9bc485cb13bf8b7b5957d74